File: wicketStart/src/main/java/de/valtech/Team.java
package de.valtech;

import org.apache.wicket.model.util.ListModel;

import java.util.List;

public class Team extends ListModel<Human> {

    Team(List<Human> team) {
        super(team);
    }

    String getFullTeamNames() {
        String output = "";
        for (Human human : this.getObject()) {
            if (output == "") {
                output += human.getFullName();
            }
            output = output + "; " + human.getFullName();
        }
        return output;
    }

}

File: react_start_task/src/Team.js

export class Team {

    constructor(teamMember){
        this.teamMember = teamMember;
        }



  getFullTeamNames(){
    let output ="";

     this.teamMember.forEach(h => {
         if (output.equals("",true)) {
             output = h.getFullName();
         }else{
         output = output+"; " + h.getFullName();
         }}
 );
    return output;
}

}